### ISSUE TITLE
Storage: 400 error on GET /storage/o/path

### ISSUE THREAD

**Issue by Vincebe** [nhost/hasura-backend-plus#438]
Hello,

When trying to get a file on `/storage/o/path` i get a 400 error.
No problem to get metadata.
HBP is installed in standalone with Minio behind Nginx (Nginx config is OK).

Error log:
```
0|hbp  | TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Last-Modified"]
0|hbp  |     at ServerResponse.setHeader (_http_outgoing.js:561:3)
0|hbp  |     at ServerResponse.header (/home/hasura-backend-plus/node_modules/express/lib/response.js:771:10)
0|hbp  |     at Object.exports.getFile (/home/hasura-backend-plus/dist/routes/storage/get.js:90:10)
0|hbp  |     at processTicksAndRejections (internal/process/task_queues.js:93:5) {
0|hbp  |   code: 'ERR_INVALID_CHAR'
0|hbp  | }
0|hbp  | ::ffff:127.0.0.1 - - [14/Mar/2021:19:49:39 +0000] "GET /storage/o/public/vehicules-autonomes.jpg HTTP/1.1" 400 40
```

After a quick check, the header was set with a timezone instead of the UTC format expected.

Modifying the following line fixes the problem.
`src/routes/storage/get.ts` (L101)
from
`res.set('Last-Modified', headObject.LastModified?.toString())`
to
`res.set('Last-Modified', headObject.LastModified?.toUTCString())`

**Comment by elitan** [comment]
Oh nice, could you send this as a PR?

**Comment by Vincebe** [comment]
Closed #439
